### PR TITLE
Add initial file related class

### PR DESCRIPTION
**Pull request by kcassia**
Add FileMgr, File, ImageFile, TextFile class

### CODE REVIEW

**Comment by hj-d** [comment]
Please do the work at the your new branch. not master..

`kcassia wants to merge 1 commit into SweetWith:master from kcassia:master`

File: src/main/java/com/sweetwith/dailynote/domain/file/FileMgr.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package com.sweetwith.dailynote.domain.file;
+
+import lombok.extern.java.Log;
+
+@Log
+public class FileMgr
+{
+    private static FileMgr instance = new FileMgr();
```
Comment (hj-d) [inline src/main/java/com/sweetwith/dailynote/domain/file/FileMgr.java]: `new FileMgr();`

Is it right?

Why create an object here?

If create here.. I think line 12 is not required.

File: src/main/java/com/sweetwith/dailynote/domain/file/File.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    private Byte[] fileBody;
+    private int fileSize;
+
+    public File(String fileName, String filePath, Byte[] fileBody, String fileAuthor)
+    {
+        this.fileName = fileName;
+        this.filePath = filePath;
+        this.fileAuthor = fileAuthor;
+        this.fileBody = fileBody;
+        fileSize = fileBody.length;
```
Comment (hj-d) [inline src/main/java/com/sweetwith/dailynote/domain/file/File.java]: `fileSize > this.fileSize`

File: src/main/java/com/sweetwith/dailynote/domain/file/ImageFile.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package com.sweetwith.dailynote.domain.file;
+
+public class ImageFile extends File
+{
+    public ImageFile(String fileName, String filePath, Byte[] fileBody, String fileAuthor)
+    {
+        super(fileName, filePath, fileBody, fileAuthor);
+        setFileType("image");
```
Comment (hj-d) [inline src/main/java/com/sweetwith/dailynote/domain/file/ImageFile.java]: Where is 'setFileType' declared?

File: src/main/java/com/sweetwith/dailynote/domain/file/FileMgr.java (hunk 1 above)
Comment (JeonJinil) [reply src/main/java/com/sweetwith/dailynote/domain/file/FileMgr.java]: When using spring container, bean is created with singleton. Modify it with that.

**Comment by kcassia** [comment]
I'll fix it

**Comment by kcassia** [comment]
I'll fix it

**Pull request closed without merging**